types/auth0/index.d.ts

```diff
@@ -1521,16 +1544,32 @@ export interface DeviceCredential {
     last_used?: string;
 }
 
+export interface sendEnrollmentTicketData {
+    user_id: string;
+    send_mail?: boolean;
+}
+
+export interface sendEnrollmentTicketResponse {
```

Comment:
```suggestion
export interface sendEnrollmentTicketResponse {
```
ditto about CamelCase notation for interfaces